site/docs/components/stepper/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Stepper
```

Comment:
Remove the old component page, to make sure users won't use the old one.
https://www.saltdesignsystem.com/salt/components/stepped-tracker/examples

Keep the main component `SteppedTracker`, for migration reasons

Reply:
Done!